js/gantt-api.js

```diff
@@ -0,0 +1,1057 @@
+(function(){
+
+var apiUrl = "https://export.dhtmlx.com/gantt";
```

Comment:
That does not sounds correct to call an external API; there certainly are limitations.

Reply:
Thanks for your feedback, I agree. But standalone modules require payed license, so I decided to use the online export service which is claimed to be free of charge on the vendor's website.
See below
https://dhtmlx.com/docs/products/dhtmlxGantt/export.shtml

Reply:
Well, some may care about privacy issues, there may be limit on the number of calls that can be done (there are many GLPI instances); ...
Also, what if the free service disapears?

Exporting should certainly be done on GLPI side. 
Adding this is not a blocker for the PR, I'm not sure current gantt implementation permit such exports (and you should keep as is on your side if you want).
But I guess removing the external call will be a blocker :)

Reply:
I don't like having data sent to an external service just for an export. At the very least, I think there needs to be a disclaimer and make it so a GLPI admin has to acknowledge that their project data will be sent to this service to be able to enable exports. Otherwise, it should be disabled.
Doing exports natively in GLPI may be possible later too.

Reply:
Would it be appropriate to have a warning message like this, until GLPI PNG/PDF exports are implemented ?
![image](https://user-images.githubusercontent.com/81109422/115583506-9f0a0d00-a2d2-11eb-9810-05c22ae78071.png)

Reply:
Hello @trasher, @cconard96,
I have removed the export function from this PR, could you please check if there is anything else to address ?
Many thanks.

Reply:
Hi, sure, I'll take another look at the code today or tomorrow.
There are still some CS issues on JS (you've fixed all PHP ones already ;))

I'm not confident with gantt, someone else will do a functionnal review.